### PR TITLE
Poland GDPR

### PR DESCRIPTION
**Pull request by baskof147**

### CODE REVIEW

File: affiliates/Poland.md [hunk 1]
```diff
@@ -93,6 +93,9 @@ This method will create or update customer and return customer uuid.
 | signature[timestamp]            | Required <br /> unix timestamp – must be UTC +/- 60 seconds                                                                                         | Timestamp when request is made                             |
 | signature[api_key]              | Required                                                                                                                                      | Api key                                                    |
 | signature[hash]                 | Required                                                                                                                                      | A sha1 concatenation of timestamp, api key and secret key  |
+| agreements.terms_of_service     | Required <br /> Integer [1 - Yes, 0 - No]                                                                                                           | Does customer agree with terms of service?                 |
```
Comment (mikk150) [inline affiliates/Poland.md]: it is not `agreements.terms_of_service` but `agreements[terms_of_service]`

File: affiliates/Poland.md [hunk 2]
```diff
@@ -93,6 +93,9 @@ This method will create or update customer and return customer uuid.
...
+| agreements.data_proccessing_policy | Required <br /> Integer [1 - Yes, 0 - No]                                                                                                        | Does customer agree with data processing policy             |
```
Comment (mikk150) [inline affiliates/Poland.md]: it is not `agreements.data_proccessing_policy` but `agreements[data_proccessing_policy]`

File: affiliates/Poland.md [hunk 3]
```diff
@@ -138,7 +141,9 @@ Body:
       "api_key":"poland",
       "timestamp":1510256918,
       "hash":"1f9158037d701039236cedc1b6efcd74bd0dc1cb"
-   }
+   },
+   "agreements.terms_of_service":"1",
```
Comment (mikk150) [inline affiliates/Poland.md]: this should be object

File: affiliates/Poland.md (hunk 1 above)
Comment (baskof147) [reply affiliates/Poland.md]: fixed

File: affiliates/Poland.md (hunk 2 above)
Comment (baskof147) [reply affiliates/Poland.md]: fixed

File: affiliates/Poland.md (hunk 3 above)
Comment (baskof147) [reply affiliates/Poland.md]: fixed

**Pull request merged**